tests/FakeItEasy.Tests.ruleset

```diff
@@ -12,7 +12,9 @@
     <Rule Id="CA1025" Action="None" />
     <Rule Id="CA1034" Action="None" />
     <Rule Id="CA1045" Action="None" />
+    <Rule Id="CA1051" Action="None" />
     <Rule Id="CA1062" Action="None" />
+    <Rule Id="CA1068" Action="None" />

```

Comment:
"CancellationToken parameters must come last". Doesn't matter in tests; violated by several scenarios in `CancellationSpecs`.